bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -211,6 +219,14 @@ private void addPrefListener() {
 
          if (property.equals(ITourbookPreferences.GRAPH_MARKER_IS_MODIFIED)) {
 
+            updateUI();
+         } else if (property.equals(ITourbookPreferences.PREF_BEVERAGECONTAINERS_HAS_CHANGED)) {
+
+            reloadTourData();
```

Comment:
`reloadTourData()` could contain also the other 2 code lines which are following, they are the same in both locations

Reply:
Fixed

Reply:
This was not fixed the way I wanted

You've put code into `updateUI` 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/d9997b2e-028e-4de5-99db-8cdb7144be9d)

It's too complicated to describe it but the new lines in updateUI must be removed because this method do not need this code.

Reply:
I think I understand now. I have made the modification but let me know if it's still not what you were asking for.

Reply:
Now you did it correctly but the red marked code needs a cleanup, it is now also in `reloadTourData()`

![image](https://github.com/mytourbook/mytourbook/assets/1283445/f651e4a3-d085-42b1-9175-39d277c681f5)